app/models/work_version.rb

```diff
@@ -89,6 +89,8 @@ class WorkTypeUpdateError < RuntimeError; end
   end
 
   def updatable?
+    return false if purl_requested? || reserving_purl?
```

Comment:
Don't the existing cases already preclude one this method from returning true if the work is in on of those states? Can you write a test?

Reply:
good point, will drop this guard clause